### PR TITLE
refactor: scss

### PR DESCRIPTION
**Pull request by Enubia**
@stacysealky fixed all the legacy scss
a few pages still need attention but 95% of the templates are now refactored

will raise tickets once I see things that need to be changed

### CODE REVIEW

**Review by stacysealky — APPROVED** [review]
LGTM but checked the deleted font once before merging

File: design/scss/_variables.scss
Comment (Enubia) [reply design/scss/_variables.scss:136]: I'll add it back but haven't found a use case for it
Maybe we use it in the future

**Pull request merged**
